corehq/apps/userreports/expressions/evaluator/functions.py

```diff
@@ -0,0 +1,20 @@
+from datetime import date, timedelta
+
+from simpleeval import DEFAULT_FUNCTIONS
+
+
+def safe_range(start, *args):
+    ret = list(range(start, *args))
+    if len(ret) < 100:
+        return ret
```

Comment:
I know this didn't change in this PR, but it would be safer if it didn't create the list until after checking the range length.
```suggestion
    ret = range(start, *args)
    if len(ret) < 100:
        return list(ret)
```